doc/source/migrations/index.rst

```diff
@@ -25,6 +25,13 @@ Development version
   ``ansys/actions/doc-deploy-changelog``, ``ansys/actions/doc-deploy-dev``, and
   ``ansys/actions/doc-deploy-stable``.
 
+**Deprecated features:**
+
```

Comment:
We are missing references to action renaming (i.e. check-pr-title). Can somebody handle this?

Reply:
Just added this in a new item list, see https://github.com/ansys/actions/pull/586/commits/716c37e854fd70d687dd95dcd4868fe45c459839